apm-sniffer/apm-agent/src/main/java/org/apache/skywalking/apm/agent/SkyWalkingAgent.java

```diff
@@ -138,6 +138,8 @@ public static void premain(String agentArgs, Instrumentation instrumentation) th
 
         Runtime.getRuntime()
                 .addShutdownHook(new Thread(ServiceManager.INSTANCE::shutdown, "skywalking service shutdown thread"));
+
+        SnifferConfigInitializer.pluginInitCompleted();
```

Comment:
Also, `Executors.newSingleThreadScheduledExecutor` is not just used in **FileWriter**, others like `JVMService` service used them too. I think put `SnifferConfigInitializer.pluginInitCompleted();` here would make more logs disappeared. You just need this at L131, right after `installOn(instrumentation)`

Reply:
> Also, `Executors.newSingleThreadScheduledExecutor` is not just used in **FileWriter**, others like `JVMService` service used them too. I think put `SnifferConfigInitializer.pluginInitCompleted();` here would make more logs disappeared. You just need this at L131, right after `installOn(instrumentation)`

Agree. I will try and verify it.